easybuild/easyconfigs/p/PLUMED/PLUMED-2.7.0-foss-2020b.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'PLUMED'
+version = '2.7.0'
+
+homepage = 'https://www.plumed.org'
+description = """PLUMED is an open source library for free energy calculations in molecular systems which
+ works together with some of the most popular molecular dynamics engines. Free energy calculations can be
+ performed as a function of many order parameters with a particular  focus on biological problems, using
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = ['https://github.com/plumed/plumed2/releases/download/v%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['b23399a3300ce3ae080e81263274ab8551e00217c3f37cb33cb7e37954fe8572']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('GSL', '2.6'),
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Boost', '1.74.0'),
+]
+
+preconfigopts = 'env FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
+configopts = '--exec-prefix=%(installdir)s --enable-gsl --enable-modules=all --enable-python i'
```

Comment:
I guess the `i` is a typo, but how come it gets ignored?

Reply:
the good old vim "i" for insertmode typo. Configure scripts aren't typically all that robust? It getting ignored doesn't surprise me much.